packages/react/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactElement, CSSProperties } from 'react';
+import '@compassion-gds/css/src/components/Table/table.css';
+
+export interface ColumnCell {
+  title: string;
+  key: string;
+  headerRenderFn?: (key: string | number, title: string) => ReactElement;
+  cellRenderFn?: (key: string, data: string | number | object) => ReactElement;
+}
+
+export interface TableProps {
+  columns: ColumnCell[],
+  rows: object[],
+  disabled?: boolean,
+  stickyHeader?: boolean,
+  containerStyle?: CSSProperties,
+  tableStyle?: CSSProperties,
+}
+
+const Table = ({ columns = [], rows = [], disabled = false, stickyHeader = false, containerStyle, tableStyle }: TableProps) => {
+  const sticky = stickyHeader ? 'sticky-header' : '';
+  const disabledTable = disabled ? 'disabled' : '';
+  const classNames = ['table', sticky, disabledTable].join(' ');
+
+  return (
+    <div className="table-container" style={containerStyle}>
+      <table className={classNames} style={tableStyle}>
+        <thead className="table-head">
+          <tr className="table-row">
+            {
+              columns.map((col, index) => (
+                col.headerRenderFn
+                  ? col.headerRenderFn(index, col.title)
```

Comment:
Why do we pass the `index` here: `col.headerRenderFn(index, col.title)` ?
We should just pass the whole `col` object, and let the function use the entire data (if it needs to) 
`col.key` can serve as unique key